### ISSUE TITLE
Treasure Hunt affected by Regen?

### ISSUE THREAD

**Issue by MikeyCarter** [orphu/mcdungeon#366]
My daughter an I wanted to do a treasure hunt... so I spawned a few in before a few regens.

Can't seem to find any instructions on how to play... all the chest just contain things you'd find in a level 1 dungeon chest... which got me thinking two things.

1.  Any way re reload a treasure hunt?   If not feature request?
2.  does a regen also redo treasure hunt chests?

And if I'm wrong on the chest being re-loaded during a regen... question, how does one start a treasure hunt .   As we've visited several sites and can't seem to find any chests with instructions books.  :(

**Comment by MikeyCarter** [comment]
Ended up pulling the co-ordinates of the other hunts.  Seems two out of the five I checked start at this small cabin with no chest... no indication what to do... think all the chests we found were the stages.

Again not sure if this is a bug or just some intended mystery we never could solve.

**Comment by sshipway** [comment]
The first location of the treasure hunt will contain a non-hidden chest that holds a book.  The book holds the clues for how to find the treasure chest.    The intermediate locations will have either nothing, or sometimes an intermediate clue book.
The level of the final treasure chest will depend on how many steps there are in the hunt.

**Comment by orphu** [comment]
Is this a bug?

**Comment by sshipway** [comment]
I don't think it is necessarily a bug; the first location chest should have the clue book and other locations may have a (non-hidden) loot chest, and possibly a hidden chest with an intermediate clue book.  The last location should have a hidden big loot chest.  

I would need to re-test everything under the latest versions though; possibly the book isn't created in the new minecraft?  I know some things like this (signs, etc) need slightly different syntax now.

@MikeyCarter : which version of minecraft are you using?

**Comment by MikeyCarter** [comment]
Would have been the latest released version of Minecraft at the time I logged this.   The problem is the first stage has no chest... thus no book.

**Comment by sshipway** [comment]
Haven't managed to duplicate this yet, except when the hunt has only 1 stage (which is a bit pointless).
The intermediate steps will of course have no visible chest (unless you have enabled intermediate loot chests) and in any case will not have clue books.  The final stage will always have a hidden chest, as may an intermediate if you have intermediate stages enabled.

It _may_ be specific to certain landmarks?  Can you let me know the landmark (cottage, memorial, well, etc) of the initial stage where there should have been a chest but wasn't?
